gulpfile.js

```diff
@@ -66,7 +67,7 @@ const SELECTED_PLATFORMS = getInputPlatforms();
 //Tasks
 //-----------------
 
-gulp.task('clean', gulp.parallel(clean_dist, clean_apps, clean_debug, clean_release));
+gulp.task('clean', gulp.parallel(clean_dist, clean_apps, clean_debug, clean_release, cordova_clean));
```

Comment:
All the other clean methods start by clean, your ends. Can you modify it to maintain the style? 
I have seen that all the Cordova methods start with Cordova but I think is better to maintain the style when possible.